### DISCUSSION TITLE
Drag and Drop Model Tabs

### DISCUSSION THREAD

**Discussion by BoxJockey** [adieyal/sd-dynamic-prompts#700] [Ideas]
Would it be possible to drag and drop the model tabs so we could easily change the order in which the models are processed?

It would be nice to be able to quickly see what differences the order of models make.

IE Tab1:Face, Tab2:Body -> Tab1:Body, Tab2:Face

EDIT: Also very useful for adding a new tab before another.

IE Tab1:Face, Tab2:Body -> Tab1:Face, Tab2:NEW TAB, Tab3:Body

**Comment by akx** [comment]
What do you mean with "the model tabs"?

**Reply by BoxJockey** [reply]
My mistake! I meant to post this to 'Adetailer' not 'Dynamic Prompts'
